bin/deploy.sh

```diff
@@ -17,23 +23,23 @@ git fetch --quiet origin 2>&1
 HIGHEST_TAG=$(git tag --list "*.*.*" | sort --version-sort | tail --lines 1)
 CURRENT_TAG=$(git describe --tags --always)
 CURRENT_BRANCH=$(git symbolic-ref --short -q HEAD)
-DIFF_TO_MASTER=$(git diff origin/master)
+DIFF_TO_BRANCH=$(git diff origin/$BRANCH)
```

Comment:
I'm not sure if it matters, but if the given branch doesn't exist then this shows a fatal error. Should we check for its existence first (with e.g. `git ls-remote origin $BRANCH`)?

Reply:
I think it is a good idea. I'll add that.